.travis.yml

```diff
@@ -32,11 +32,17 @@ jobs:
       before_script: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.23.3
       script: ./scripts/verify.sh
 
-    - stage: golden
+    - stage: generate
       # The golden_test.sh check if the the testdata is updated according to the current changes
       # To update the testdata use the Makefile targets `make generate-setup` then `make generate-testdata`
       script: ./golden_test.sh
```

Comment:
```suggestion
      script: make generate
```

Reply:
The `make generate` just update and not check if it was updated or not which is the goal of this job.